deep_quoridor/src/train_alphazero.py

```diff
@@ -77,23 +79,27 @@ def train_alphazero(
             new_replay_buffer_items = self_play_manager.get_results(timeout=0.1)
         training_agent.replay_buffer.extend(new_replay_buffer_items)
         self_play_manager.join()
+        game_num = (epoch + 1) * args.games_per_epoch
+
+        Timer.finish("self-play", game_num)
 
         # Do training if we have enough samples in the replay buffer.
         # training_agent.episode_count = game_num
         training_occured = training_agent.train_iteration(epoch=epoch)
         if not training_occured:
             print("Not enough samples - skipping training")
 
-        game_num = (epoch + 1) * args.games_per_epoch
         current_filename = training_agent.save_model_with_suffix(f"_epoch_{epoch}")
-        if wandb_train_plugin is not None:
+        if wandb_train_plugin is not None and epoch < args.epochs - 1:
```

Comment:
This is an unrelated change, it was computing the tournament here for the last epoch, and then again below when doing end_arena, so I removed this one, since the other does a few more things like uploading the models